torch/testing/_internal/distributed/nn/api/remote_module_test.py

```diff
@@ -208,6 +211,25 @@ def test_remote_parameters(self):
             self.assertEqual(len(param_rrefs), 1)
             self.assertTrue(torch.equal(param_rrefs[0].to_here(), _PARAM_VAL))
 
+    @dist_utils.dist_init
```

Comment:
You can just use the decorator `@skip_if_lt_x_gpu(1)` to run the test only on GPUs.